zulipterminal/ui_tools/buttons.py

```diff
@@ -95,6 +100,12 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         else:  # This is in the else clause, to avoid multiple activation
             return super().keypress(size, key)
 
+    def set_status(self, is_active: bool) -> None:
+        if is_active:
+            self._w.set_attr_map({None: "active_narrow"})
+        else:
+            self._w.set_attr_map({None: None})
```

Comment:
This works, but one of the issues we discussed in the stream is how this interacts with other changes where the button is updated, due to line 84. As I think Rohitth mentioned, this means we likely need state in the button, so it knows the correct behavior to do there.